pkg/controllers/user/logging/utils/wrap.go

```diff
@@ -226,6 +247,16 @@ func getDateFormat(dateformat string) string {
 	return "%Y.%m.%d"
 }
 
+func testReachable(network string, url string) error {
+	timeout := time.Duration(10 * time.Second)
+	conn, err := net.DialTimeout(network, url, timeout)
+	if err != nil {
+		return fmt.Errorf("url %s unreachable, error: %v", url, err)
+	}
+	defer conn.Close()
```

Comment:
Nit: what is the point of `defer` here? given that there is no other execution after this line, only a return, just using conn.Close() should be enough